mirrord/config/src/lib.rs

```diff
@@ -42,7 +42,7 @@ use crate::{
 /// skip_processes = ["ide-debugger", "ide-service"] # we don't want mirrord to hook into these
 ///
 /// [agent]
-/// log_level = "debug"
+/// log_level = "trace"
```

Comment:
This change was a search/replace mistake, oops.

Reply:
```suggestion
/// log_level = "debug"
```